packages/ERTP/src/issuer.js

```diff
@@ -9,225 +8,11 @@ import { isPromise } from '@agoric/produce-promise';
 
 import makeAmountMath from './amountMath';
 
-/**
- * @typedef {import('./amountMath').Amount} Amount
- * @typedef {import('./amountMath').Value} Value
- * @typedef {import('./amountMath').AmountMath} AmountMath
- * @typedef {Payment|Promise<Payment>} PaymentP
- */
-
-/**
- * @typedef {Object} Issuer
```

Comment:
I see types such as `Issuer` got moved to `types.js`; how is the reader of `issuer.js` expected to discover this? Just by convention? I'd prefer an explicit import, but I guess that's just not how this works. A 1-line comment seems cost-effective.

Reply:
Highlight the type `Issuer` and hit F12 in VSCode/Codium.

Reply:
A 1-line comment still seems cost-effective for readers not using an IDE... for example, browsing via github.

Reply:
How would you word this/where you'd like to put this?  In every contract?  In all the Zoe/ERTP implementation files?

If you have a concrete suggestion, I'd be happy to implement it.

Reply:
The `@returns {IssuerKit}` bit of jsdoc implicitly refers to `types.js`. I suggest making that explicit:

```
@returns {IssuerKit} see types.js
```

It seems particularly relevant in this case where dozens of lines of documentation were moved elsewhere without a trace. But ideally, I'd like the 1st such implicit reference in each file made explicit.

I guess another possibility is a paragraph or section in CONTRIBUTING.md about the types.js idiom.

Reply:
Okay.  Thinking it over, I'll do the explicit import (`import './types'`).  That will reduce drift from what the file actually uses and any renamings of types files.